openqa-clone-and-monitor-job-from-pr

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env perl
+# Copyright SUSE LLC
+
+=head1 NAME
+
+openqa-clone-and-monitor-job-from-pr
+
+=head1 SYNOPSIS
+
+Clones and monitors openQA jobs mentioned in a PR description as CI action. This
+script is supposed to be used via the GitHub action defined in
+C<actions/clone-job/action.yaml> like this:
+
+=begin text
+---
+name: Run openQA tests
+on:
+  pull_request_target:
+  workflow_dispatch:
+env:
+  OPENQA_HOST: ${{ secrets.OPENQA_URL }}
+  OPENQA_API_KEY: ${{ secrets.OPENQA_API_KEY }}
+  OPENQA_API_SECRET: ${{ secrets.OPENQA_API_SECRET }}
+  GH_REPO: ${{ github.event.pull_request.head.repo.full_name }}
+  GH_REF: ${{ github.event.pull_request.head.ref }}
+  GH_PR_BODY: ${{ github.event.pull_request.body }}
+jobs:
+  clone_and_monitor_job_from_pr:
+    steps:
+      - uses: os-autoinst/scripts/actions/clone-job@master
+=end text
+
+It will then clone and monitor an openQA job when a PR mentioning one via e.g.
+C<@openqa: Clone https://openqa.opensuse.org/tests/123456> is created. By
+default it will clone the job on o3 into the "Development / GitHub" group. To
+use a different openQA instance and group, set the environment variables
+C<OPENQA_HOST> and C<OPENQA_SCHEDULE_GROUP_ID> accordingly.
+
+For local testing you may also invoke the script manually. Have a look at the
+handling of environment variables at the beginning of the script code as you
+need to set certain additional environment variables that are normally supplied
```

Comment:
```suggestion
For local testing you may also invoke the script manually. Have a look at the
handling of environment variables at the beginning of the script code as you
need to set certain additional environment variables, the `test/05-clone-and-monitor.t`
can also clarify this script's behavior from user perspective.
```

Since this is only meant for GH, I think it's ok to redirect the reader to how the test is done.